wormhole-connect/src/hooks/useIsTransferLimited.ts

```diff
@@ -105,14 +105,12 @@ const useIsTransferLimited = (): IsTransferLimitedResult => {
       (async () => {
         for (const rpcHost of GOVERNOR_API_BASE_URLS) {
           try {
-            const baseUrl = `${rpcHost}${
-              rpcHost.endsWith('/') ? '' : '/'
-            }v1/governor`;
+            const baseUrl = `${rpcHost}${rpcHost.endsWith('/') ? '' : '/'}`;
             const [tokenListResponse, availableNotionalByChainResponse] =
               await Promise.all([
-                axios.get<TokenList>(`${baseUrl}/token_list`),
+                axios.get<TokenList>(`${baseUrl}v1/governor/token_list`),
                 axios.get<AvailableNotionalByChain>(
-                  `${baseUrl}/available_notional_by_chain`,
+                  `${baseUrl}api/v1/governor/limit`,
```

Comment:
rpcHost could be wormholescan or public guardian api, and `api/v1/governor/limit` is not compatible with guardian api, check here for instance [https://wormhole-v2-mainnet-api.mcf.rocks/api/v1/governor/limit](https://wormhole-v2-mainnet-api.mcf.rocks/api/v1/governor/limit). The fix should come from wormholescan API